src/nodes/corenodes/input/noise_image_node/noise_image_node.py

```diff
@@ -50,27 +51,45 @@ def NodeDndEventHook(self):
         self.RefreshNodeGraph()
 
     def NodeInitProps(self):
-        noise_seed = api.PositiveIntegerProp(
-            idname="noise_seed",
-            default=1,
-            min_val=0,
-            max_val=100,
-            fpb_label="Noise Seed"
+        height = api.IntegerProp(
+            idname="height",
+            default=512,
+            min_val=1,
+            max_val=10000,
+            show_p=True,
+            fpb_label="Height"
         )
 
-        self.NodeAddProp(noise_seed)
+        width = api.IntegerProp(
+            idname="width",
+            default=512,
+            min_val=1,
+            max_val=10000,
+            show_p=True,
+            fpb_label="Width"
+        )
+        self.NodeAddProp(height)
+        self.NodeAddProp(width)
```

Comment:
Please flip these properties around so that width appears above height (also make sure to shift lines 54 - 61 below lines 63 - 70).

```suggestion
        self.NodeAddProp(width)
        self.NodeAddProp(height)
```